contrib/gorilla/mux/option.go

```diff
@@ -103,3 +106,32 @@ func WithResourceNamer(namer func(router *Router, req *http.Request) string) Rou
 		cfg.resourceNamer = namer
 	}
 }
+
+// WithHeaderTags specifies that the integration should attach HTTP request headers as
+// tags to spans.
+// Warning: using this feature can risk exposing sensitive data such as authorisation tokens
+// to Datadog.
+func WithHeaderTags() RouterOption {
+	return func(cfg *routerConfig) {
+		cfg.headerTags = true
+	}
+}
+
+func headerTagsFromRequest(req *http.Request) ddtrace.StartSpanOption {
```

Comment:
Please move this into `gorilla/mux/mux.go` where it is used. It's out of place here.